public/static/css/common.css

```diff
@@ -103,10 +103,6 @@ body {
     border-color: #adadad;
 }
 
-button > i + span {
-	margin-left: 5px;
-}
```

Comment:
While `common.css` has been altered before, initially it was though to contain global CSS definitions from dlang.org (might do a cleanup later).